lib/logs.js

```diff
@@ -124,6 +125,10 @@ module.exports = {
 	},
 
 	functionLogsShowLogs(logStreamNames) {
+		if (logStreamNames.length === 0) {
```

Comment:
Use lodash's `_.isEmpty(logStreamNames)` here. This will also work for cases where logStreamNames might be nil.